obgyn_config/models.py

```diff
@@ -12,9 +18,13 @@ class ObgynConfigModel(models.Model):
     rs_per_room = models.FloatField(default=0, null=True)
     operative_charge = models.FloatField(default=0, null=True)
     rs_per_day_nursing = models.FloatField(default=0, null=True)
+    field_master_name  = models.ForeignKey(FieldMasterModel, on_delete=models.DO_NOTHING, null=True)
+    city_id = models.ForeignKey(CityModel, on_delete=models.CASCADE, null=True)
+    taluka_id = models.ForeignKey(TalukaModel, on_delete=models.CASCADE, null=True)
+    district_id = models.ForeignKey(DistrictModel, on_delete=models.CASCADE, null=True)
+    state_id = models.ForeignKey(StateModel, on_delete=models.CASCADE, null=True)
```

Comment:
Change every foreign key names. remove _id from foreign keys as it is not required.